Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -703,7 +703,10 @@ Root
             XCTAssertNil(navigatorIndex.path(for: 0)) // Root should have not path persisted.
             XCTAssertEqual(navigatorIndex.path(for: 1), "/documentation/fillintroduced")
             XCTAssertEqual(navigatorIndex.path(for: 4), "/tutorials/testoverview")
-            XCTAssertEqual(navigatorIndex.path(for: 10), "/documentation/fillintroduced/maccatalystonlydeprecated()")
+            XCTAssertEqual(navigatorIndex.path(for: 9), "/documentation/fillintroduced/maccatalystonlydeprecated()")
```

Comment:
The indices if various items changed when the UncuratedClass entered the navigator hierarchy.